appengine/storage.py

```diff
@@ -72,10 +74,19 @@ def keyToXml(key_provided):
   client = ndb.Client()
   with client.context():
     result = Xml.get_by_id(key_provided)
+
   if not result:
     xml = ""
   else:
     xml = result.xml_content
+    # Update row to include last accessed time--because last_accessed is set to
+    # auto_now, any write updates the time.
+    with client.context():
+      row = Xml(id = key_provided,
+        xml_hash = result.xml_hash,
+        xml_content = result.xml_content)
+      row.put();
```

Comment:
would it work if you just did `result.put()` (instead of creating/copying result)?

What's the benefit of `auto_now` vs explicitly updating the `last_accesed` time here? It seems more straightforward to me if we just updated `last_accessed`.

Reply:
Switched to just doing `result.put()`.

I like using `auto_now` because it ensures consistency in datetime formatting, timezones, etc.

Reply:
I tested locally with `dev_appserver.py`. I still haven't been able to view the datastore directly, but I verified that the last updated time is getting updated correctly on read.